src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -606,6 +633,11 @@ export default function DeploymentTemplateOverrideForm({
                     convertVariables={convertVariablesOverride}
                     setConvertVariables={setConvertVariables}
                     groupedData={groupedData}
+                    hideLockedKeys={hideLockedKeys}
+                    lockedConfigKeysWithLockType={lockedConfigKeysWithLockType}
+                    hideLockKeysToggled={hideLockKeysToggled}
+                    removedPatches={removedPatches}
+                    selectedTabIndex={state.selectedTabIndex}
```

Comment:
We should use it from the context instead of passing

Reply:
done